solidity/dashboard/src/components/CircularProgressBar.jsx

```diff
@@ -112,7 +116,11 @@ export const CircularProgressBars = React.memo(
           </g>
         </svg>
         <div className="mb-1">
-          {withLegend && items.map(renderProgressBarLegendItem)}
+          <ProgressBarLegendContext.Provider
+            value={{ displayLegendValuFn: displayAmountWithMetricSuffix }}
```

Comment:
I believe this, and its derivatives in `ProgressBar.jsx`, should be `displayLegendValueFn`, unless that's taken for some reason?

Reply:
Here we wrap a legend section in `<ProgressBarLegendContext />` because component renders `ProgressBarLegendItem` which uses a context.

Reply:
I was obviously so tired I wasn't doing a good job of pointing out what I wanted to at this point haha. The note here was referring to a spelling issue, the missing `e` in `Value` in `displayLegendValueFn`.

Reply:
sorry too 😆. I was out of brain power and thought we were wondering about why there is a `value={{ displayLegendValuFn: fn }}`instead of `<ProgressBarLegend displayAMuntFn={fn} />`